examples/single-account-ecs/variables.tf

```diff
@@ -136,3 +136,22 @@ variable "tags" {
     "product" = "sysdig-secure-for-cloud"
   }
 }
+
+#
+# Autoscaling configurations
+#
+variable "enable_autoscaling" {
+  type        = bool
+  description = "Whether to enable autoscaling or not"
+}
```

Comment:
i would advocate for examples to have clear default boolean values

Reply:
👍